SU2_CFD/src/output/CFlowIncOutput.cpp

```diff
@@ -396,6 +416,10 @@ void CFlowIncOutput::SetVolumeOutputFields(CConfig *config){
     break;
   }
 
+  if (config->GetKind_Species_Model() != SPECIES_MODEL::NO_SCALAR_MODEL) {
+    AddVolumeOutput("SPECIES", "Species", "SOLUTION", "Species mass fraction");
+  }
```

Comment:
It's a bit confusing that in places you deal with multiple species but in others with only one

Reply:
Yep, The output needs to be adapted to handle and arbitrary number of species. Currently the code can handle multiple species in some places and in others not. I will have to go over all this with a suitable Testcase and make it work 👍

Reply:
This is taken care for in 6963819 8282034